### ISSUE TITLE
Bug in fast-forwarding of MP4 video format.

### ISSUE THREAD

**Issue by dxpele** [iina/iina#5053]
An MP4 video with fast forward issues, where every time fast forward is used, the video resets.

**Comment by hugepower** [comment]
I believe this issue is usually due to errors in video encapsulation. For example, when using tools like [Streamlink](https://github.com/streamlink) to record live streams, it occasionally happens that the recorded files cannot fast forward or rewind properly. Re-encapsulating the video typically resolves these issues.

**Comment by low-batt** [comment]
@hugepower is correct. This can be caused by how the video is encoded. However we should rule out a known problem.

Does the behavior in [this comment](https://github.com/iina/iina/issues/4502#issuecomment-1656492860) describe what you are experiencing?

If so, try applying the workaround described at the end of [this comment](https://github.com/iina/iina/issues/4502#issuecomment-1986987316) and report if that changes the behavior.

A [MP4](https://en.wikipedia.org/wiki/MP4_file_format) is a container format that can hold video streams. When playing a video click on `Inspector` under the `Window` menu. In the `Inspector` window that opens look for `Format`, the first label in the `VIDEO` section. This lists the codec that was used to encode the video. That is useful to know in this case.

**Comment by hugepower** [comment]
@low-batt 

Yes, I’ve personally experienced similar issues. In such cases, I typically resolve them using the following command:

```shell
ffmpeg -i file.mp4 -c:v copy -c:a copy new_file.mp4
```

## Relevant Context

[Timestamp error messages when joining MP4s with ffmpeg](https://forum.videohelp.com/threads/374836-Timestamp-error-messages-when-joining-MP4s-with-ffmpeg#post2415452)

**Comment by low-batt** [comment]
I'm not sure re-encoding will fix the seeking problem in issue #4502 as that was traced to a FFmpeg regression and reported in [Ticket #10585](https://trac.ffmpeg.org/ticket/10585). Video files that used to work fine exhibit problems with relative seeking after IINA upgraded FFmpeg. The workaround forces use of exact seeking.

**Comment by low-batt** [comment]
@dxpele Does issue #4052 match what you are experiencing?

**Comment by low-batt** [comment]
The expectation is that this issue is a duplicate of issue #4052. The root cause of that problem was a regression in FFmpeg. The development version of IINA has been upgraded to mpv 0.41.0 and FFmpeg 9.0.1 (see PR #6261). The new FFmpeg libraries contain the fix for #4052. I have tested the latest IINA nightly build and confirmed that seeking no longer causes playback to stop or restart from the beginning. This fix will be included in the IINA 1.5.0 release.

IINA is now following GitHub's practice of closing issues when they are fixed in the development version of IINA.
